crops/command_line/crops-renumber.py

```diff
@@ -73,9 +76,14 @@ def main():
                 try:
                     newstructure = cop.renumber_pdb(seqset[seqname], structure)
                 except (AttributeError, IndexError) as e:
-                    logger.warning('Something has gone wrong during renumbering:\n{}'.format(e))
-                    logger.info('Attempting Needleman-Wunsch...')
-                    newstructure = cop.renumber_pdb_needleman(seqset[seqname], structure)
+                    if args.force_alignment is True:
```

Comment:
Does it make sense to use `is` here? Is there any reason why this variable wouldn't be a boolean? Considering this comes straight from `argparse` I would suggest to use `if args.force_alignment:` instead.